backends/advanced/src/advanced_omi_backend/database.py

```
@@ -520,6 +520,38 @@ async def get_sessions_with_speech(self, user_id: str, limit: int = 100):
 
         return await cursor.to_list(length=None)
 
+    async def update_transcription_status(
```

Ruff (0.13.1)

Redefinition of unused `update_transcription_status` from line 389

(F811)